openedx/core/lib/tests/test_request_utils.py

```diff
@@ -219,91 +219,90 @@ class CustomError2(Exception):
-        self, expected_errors_setting, mock_logger, mock_set_custom_attribute,
+    def test_process_exception_with_empty_ignored_errors(
+        self, ignored_errors_setting, mock_logger, mock_set_custom_attribute,
     ):
-        with override_settings(EXPECTED_ERRORS=expected_errors_setting):
-            ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+        with override_settings(IGNORED_ERRORS=ignored_errors_setting):
+            IgnoredErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
 
         mock_logger.info.assert_not_called()
         mock_set_custom_attribute.assert_not_called()
 
-    @override_settings(EXPECTED_ERRORS=[{
+    @override_settings(IGNORED_ERRORS=[{
         'MODULE_AND_CLASS': 'test.module.TestException',
-        'REASON_EXPECTED': 'Because',
+        'REASON_IGNORED': 'Because',
     }])
     @patch('openedx.core.lib.request_utils.set_custom_attribute')
     @patch('openedx.core.lib.request_utils.log')
-    def test_process_exception_not_matching_expected_errors(self, mock_logger, mock_set_custom_attribute):
-        ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+    def test_process_exception_not_matching_ignored_errors(self, mock_logger, mock_set_custom_attribute):
+        IgnoredErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
 
         mock_logger.info.assert_not_called()
-        mock_set_custom_attribute.assert_called_once_with('checked_error_expected_from', 'middleware')
+        mock_set_custom_attribute.assert_called_once_with('checked_error_ignored_from', 'middleware')
 
-    @override_settings(EXPECTED_ERRORS=[
+    @override_settings(IGNORED_ERRORS=[
         {
             'MODULE_AND_CLASS': 'test.module.TestException',
-            'REASON_EXPECTED': 'Because',
+            'REASON_IGNORED': 'Because',
         },
         {
             'MODULE_AND_CLASS': 'openedx.core.lib.tests.test_request_utils.CustomError1',
-            'REASON_EXPECTED': 'Because',
+            'REASON_IGNORED': 'Because',
         }
     ])
     @patch('openedx.core.lib.request_utils.set_custom_attribute')
     @patch('openedx.core.lib.request_utils.log')
-    def test_process_exception_expected_error_with_defaults(self, mock_logger, mock_set_custom_attribute):
-        ExpectedErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
+    def test_process_exception_ignored_error_with_defaults(self, mock_logger, mock_set_custom_attribute):
+        IgnoredErrorMiddleware('mock-response').process_exception(self.mock_request, self.mock_exception)
 
         mock_logger.info.assert_not_called()
         mock_set_custom_attribute.assert_has_calls(
             [
-                call('checked_error_expected_from', 'middleware'),
-                call('error_expected', True),
```

Comment:
1. Because this test was for expected errors, it might go away.
2. However, if this test was also testing anything that wasn’t tested anywhere else, we might need to move those assertions.

Reply:
@robrap what should we want to achieve here should we need to remove `call('checked_error_expected_from', 'middleware'),` this one, or just need to rename it like this `call('checked_error_ignored_from', 'middleware'),` in` openedx/core/lib/tests/test_request_utils.py` 289 lines of the above comment.

Reply:
I see. I think this test is ok (wasn't duplicated), and I think the rename to `checked_error_ignored_from` looks good. I would mark this as resolved, but it looks like it was already marked as resolved. 

In the future, we should probably leave a final comment when resolving threads about why or how it was resolved.

Reply:
@robrap sure we will. Thanks